### DISCUSSION TITLE
How to bind Ctrl+Shift+0 to ActivateTab(0)

### DISCUSSION THREAD

**Discussion by ferdinandyb** [wezterm/wezterm#4401] [General]
I'm using a Hungarian keyboard layout. I want to configure wezterm so that most of the shortcuts are `Ctrl+Shift+something`. I already disabled the default keybindings, because for example `+` on my layout is `shift+3` and it seems to me `ctrl+shift+3` was understood as `ctrl++` i.e increase font size. Would I need to check what every `shift+number` produces and bind that with `ctrl+symbol`?

**Comment by danjessen** [comment]
You should probably look into https://wezfurlong.org/wezterm/config/keys.html#configuring-key-assignments

The phys vs mapped keys

**Comment by wez** [comment]
Shifted symbols can be confusing.  Take a look at https://wezfurlong.org/wezterm/troubleshooting.html#debugging-keyboard-related-issues to see how to log the key events which will help you figure out exactly what you want to bind to.

**Comment by danjessen** [comment]
At least wezterm makes it easy to unload all keybinds. Other apps just have nonworking / confusing shortcuts when you are on ISO layouts
